ROADMAP.md

```diff
@@ -28,4 +8,34 @@
  
 ## Other features we are working on in the 1.7 timeframe
 
+* etcd controller to allow moving between versions
 * kops server for better team scenarios
+* support for bare-metal
+* more gossip backends
```

Comment:
S3 is now the only piece left.  If we could replace that with something gossip, that would be useful.